src/phylum/ci/git.py

```diff
@@ -45,6 +46,29 @@ def git_remote(git_c_path: Optional[Path] = None) -> str:
     return remote
 
 
+def git_fetch(repo: Optional[str] = None, ref: Optional[str] = None, git_c_path: Optional[Path] = None) -> None:
+    """Execute a `git fetch` command with optional repository and refspec specified.
+
+    See git documentation for more detail: https://git-scm.com/docs/git-fetch
+
+    The optional `git_c_path` is used to tell `git` to run as if it were started in that
+    path instead of the current working directory, which is the default when not provided.
+    """
+    base_cmd = git_base_cmd(git_c_path=git_c_path)
+    cmd = [*base_cmd, "fetch"]
+    if repo is not None:
```

Comment:
~~Repo might be a bit confusing here. We're talking about remotes not repos here, right?~~

Surprisingly, the `git fetch` CLI itself calls this parameter a `repository`. It's not wrong, though it is a little surprising. So I think it should be fine.

Reply:
I was also a bit thrown by the naming. There was a thought that others might be as well, so the docstring for the function was updated in a later commit to add a reference to the git command's documentation.